src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -11,7 +11,8 @@
                 <id>proxy.general.icpPort</id>
                 <label>ICP port</label>
                 <type>text</type>
-                <help>The port number where Squid sends and receives ICP queries to and from neighbor caches. Leave blank to disable (default). The standard UDP port for ICP is 3130.</help>
+                <help>The port number where Squid sends and receives ICP queries to and from neighbor caches. 
+                    Leave blank to disable (default). The standard UDP port for ICP is 3130.</help>
```

Comment:
Merge with line above. XML will add lots of spaces here...

Reply:
Was not ready yet with editing. I am not really experienced with Git. When editing a file in online mode which is already part of a pull request, will automatically trigger a new pull request?

Reply:
No rush, if you need more time or help just let me know. Online edit on this branch should create new commits for this PR.